build.py

```diff
@@ -16,12 +16,17 @@
 OUTPUT_FORMATS = ['pdf', 'epub', 'html']
 
 
-def clone_or_update_repo(module_name, branch):
+def clone_or_update_repo(module_name, branch, is_production=False):
     """
     Clone or update a repository based on the module name and branch provided.
     If the repository already exists, update it. Otherwise, clone the repository.
     """
-    repo_url = f'https://github.com/openwisp/{module_name}.git'
+    if is_production:
```

Comment:
I am not convinced this approach is comfortable.
This is the standard case right? So I think we should default to this case, without the need of having to set any env variable.
The other case of using the git protocol and SSH, is the special case we need to be aware of.
So we could require setting the env var for that case as "DEV=1". We need to document this in the README.